### PR TITLE
update formula with map.jinja and style guide references, improve REA…

### PR DESCRIPTION
**Pull request by aboe76**
Update map.jinja with better merging.
Update README for style guide references
Updated VERSION and CHANGELOG file

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # should contain the currently released version of the formula
-1.0.0
+0.1.0
```
Comment (myii) [inline VERSION]: `0.1.1` to match the value in [`CHANGELOG.rst`](https://github.com/saltstack-formulas/template-formula/pull/20/files#diff-de0a9b67ffe2874a076b5fdb15bad484R4).

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,31 @@
 template formula
```
Comment (myii) [inline CHANGELOG.rst]: Excellent, tagging releases has already started!  May I recommend standardising our changelogs according to the format set out at https://keepachangelog.com/en/1.0.0/?

Comment (aboe76) [reply CHANGELOG.rst]: I pushed the tags myself. :)

Comment (aboe76) [reply CHANGELOG.rst]: thats on the todo list...I hope....

File: VERSION (hunk 1 above)
Comment (aboe76) [reply VERSION]: yeah but I need to have a commit hash to make a tag.. I think........

File: CHANGELOG.rst (hunk 2 above)
Comment (myii) [reply CHANGELOG.rst]: @aboe76 Yes, I've already pulled and appreciated the tags in `gitk`!  Looks great, thanks for running with that.

File: VERSION (hunk 1 above)
Comment (myii) [reply VERSION]: But when this is merged, it will _be_ `v0.1.1`.

**Pull request merged**